### PR TITLE
fix link to readme md --> rst

### PR DESCRIPTION
**Pull request by Pack3tL0ss**
I'm not overly familiar with sphinx, suspect this file may be auto-generated.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -7,7 +7,7 @@ Welcome to python-zeroconf documentation!
 
 .. image:: https://img.shields.io/pypi/v/zeroconf.svg
     :target: https://pypi.python.org/pypi/zeroconf
-   
+
```
Comment (jstasiak) [inline docs/index.rst:10]: Unrelated whitespace change, please restore this.

**Review by jstasiak — APPROVED** [review]
Thank you!

**Review by jstasiak — CHANGES_REQUESTED** [review]
I have only one request – see above.

File: docs/index.rst (hunk 1 above)
Comment (Pack3tL0ss) [reply docs/index.rst]: Done, my ide removed it automatically on save

**Comment by jstasiak** [comment]
Thanks!
